randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1246,12 +993,12 @@ Extra - asset_id: collision_camera_015
 
 > Aiming Platform; Heals? False
   * Layers: default
+  > Door to Twin Robot Arena
+      Trivial
   > Door to Space Jump Room (Mid)
       After Ferenia - Storm Missile Gate for SJ
   > Door to Space Jump Room (Top)
       Trivial
-  > Tile Group (POWERBEAM) 1
-      Trivial
   > Event - Storm Missile Gate
       Activate Storm Missile Locks
```

Comment:
Nothing wrong with this, but technically this node isn't needed anymore. Could just connect to one of the doors.